src/resources/css/tribe-events-full.css

```diff
@@ -147,9 +147,9 @@ ul.tribe-events-sub-nav a:hover,
 	background-image: none;
 	border: 0;
 	-webkit-border-radius: 3px;
-	        border-radius: 3px;
+			border-radius: 3px;
```

Comment:
While we use tabs religiously, when aligning by character placement (e.g. aligning `border-radius` with the similar part of the property name in `-webkit-border-radius`), spaces should be used to ensure consistent alignment across various editors/tab configurations.

GitHub makes this obvious here, where the `border-radius` is no longer aligned.